styles/global/_vars.scss

```diff
@@ -26,6 +26,11 @@ $color-gray-medium:          #757575; // lightest gray that passes color contras
 $color-gray-light:           #aeb0b5; // lighten($color-gray-dark, 60%)
 $color-gray-lighter:         #d6d7d9; // lighten($color-gray-dark, 80%)
 $color-gray-lightest:        #f1f1f1; // lighten($color-gray-dark, 91%)
+$color-gray-border:          #D8D8D8;
+$color-base:                 #555;
+
+$br-color-primary:              #0071bc;
```

Comment:
Assuming the `br-` is for Broker, I had removed these since Broker can use the same colors (`$color-primary` and `$color-primary-darker`)

Reply:
aaah got it!